src/app/features/cases/hooks/useCaseByUUID.ts

```diff
@@ -0,0 +1,19 @@
+import { useEffect } from "react"
+
+import { useGlobalState, useGlobalActions } from "app/state/state/globalState"
+
+/**
+ * Returns data, and kicks of an index onMount.
+ */
+export const useCaseByUUID = (uuid: API.Case["uuid"]) => {
+  const { cases: { index } } = useGlobalActions()
+  const { cases: { isFetching, data } } = useGlobalState()
+
+  useEffect(() => {
+    index()
+  }, [index])
```

Comment:
This is now handled in https://github.com/Amsterdam/zaken-frontend/pull/80

Reply:
OK. Will remove